core/lib/multivm/src/tracers/call_tracer/metrics.rs

```diff
@@ -0,0 +1,15 @@
+use vise::{Buckets, Histogram, Metrics};
+
+#[derive(Debug, Metrics)]
+#[metrics(prefix = "call_tracer")]
```

Comment:
Bikeshedding: It may make sense to include the high-level component name (`vm`?) first – `vm_call_tracer`.